integration

```diff
@@ -116,6 +116,7 @@
   "blear/hasslife",
   "blindlight86/HA_USR-R16",
   "bmcclure/ha-aquanta",
+  "Bobsilvio/calcio-live"
```

Comment:
This is invalid JSON.

Reply:
sorry I don't understand what I have to do

Reply:
missing comma (`,`)

Reply:
ok update